### PR TITLE
feat: Config option to set default protocol to https

### PR DESCRIPTION
**Pull request by pxlrbt**
When running `expose` without any options it tries to detect some Valet configuration and defaults to `http://[FOLDER].[DEFAULT_TLD]`. On some systems HTTPS may be the standard for sites, so I added an option to define whether HTTPS should be used as an default.

### CODE REVIEW

**Comment by mpociot** [comment]
Thanks for the PR! Can you fix the conflicts?

**Comment by pxlrbt** [comment]
@mpociot I merged the latest changes.

**Comment by pxlrbt** [comment]
@mpociot Any feedback on this?

File: expose [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env php
+#!/usr/bin/env /usr/sbin/php
```
Comment (mpociot) [inline expose:1]: Why did you add this change?

Comment (pxlrbt) [reply expose]: I thought it was a merge with the master branch, but actually I have no idea where this was coming from. I reverted that commit.

**Comment by pxlrbt** [comment]
@mpociot Anything I could help with to get this merged?

**Comment by mpociot** [comment]
Sorry that this took a bit longer than expected, but thank you for the work on this nonetheless!

**Comment by pxlrbt** [comment]
Nice. Thanks for merging.
